npm/react/plugins/react-scripts/reactScriptsFive.ts

```diff
@@ -0,0 +1,59 @@
+import _debug from 'debug'
+import type { Configuration } from 'webpack'
+import reactScriptsPackageJson from 'react-scripts/package.json'
+
+const debug = _debug('@cypress/react:react-scripts')
+
+type DefinePlugin =
+  | { definitions: Record<string, Record<string, any>> }
+  | undefined;
+type ESLintWebpackPlugin =
+  | { options: { baseConfig?: { globals?: Record<string, any> } } }
+  | undefined;
+
+export function reactScriptsFiveModifications (webpackConfig: Configuration) {
+  // React-Scripts sets the webpack target to ["browserslist"] which tells
+  // webpack to target the browsers found within the browserslist config
+  // depending on the environment (process.env.NODE_ENV). Since we set
+  // process.env.NODE_ENV = "test", webpack is unable to find any browsers and errors.
+  // We set BROWSERSLIST_ENV = "development" to override the default NODE_ENV search of browsers.
+  process.env.BROWSERSLIST_ENV = 'development'
```

Comment:
should we check to see if BROWSERSLIST_ENV  is already defined and use that in case the dev is purposefully adding the envar on their own?

Reply:
Couldn't hurt, I'll add the check